### ISSUE TITLE
Link to Code of Conduct

### ISSUE THREAD

**Issue by badboy** [Rustaceans/rust-cologne#15]
Given that we already act under the [Rust Code of Conduct](https://www.rust-lang.org/en-US/conduct.html), we should have an explicit link to it.
Alternatively, we can adopt the [Berlin CoC](http://berlincodeofconduct.org/), which is especially tailored to events.

**Comment by colin-kiegel** [comment]
Good idea. I prefer the Rust CoC over the Berlin one because I think it is less talking and more to the point. It just immediately starts with the expected behaviour.

**Comment by killercup** [comment]
I'm fine with both. Just putting a link in the footer is already a great improvement.

If I were to choose though, I'd prefer the Berlin CoC as it is already tailored to events and we don't need to rewrite it to fit or needs. Additionally, I'd appreciate it if we could link to/include one specific version. I'd suggest adding a subpage to rust.cologne that includes [the current version of the text](https://github.com/rubyberlin/code-of-conduct/blob/95e0e4460f458c35717445f28d29973160c89003/source/localizable/_code-of-conduct.en.md) mentioning that it is the Berlin CoC (and linking to berlincodeofconduct.org). (The text itself is licensed as Creative Commons Attribution-ShareAlike.)
